src/main/java/io/kuzzle/sdk/core/Collection.java

```diff
@@ -945,6 +961,324 @@ public void getMapping(final Options options, @NonNull final ResponseListener<Co
   }
 
   /**
+   * Create the provided documents
+   *
+   * @param documents Array of documents to create
+   * @param options Optional parameters
+   * @param listener Response callback
+   * @return Collection kuzzle data collection
+   */
+  public Collection mCreate(final JSONArray documents, final Options options, @NonNull final ResponseListener<JSONObject> listener) throws JSONException {
```

Comment:
Unless I'm mistaken, function prototypes without a listener are missing (write actions does not require a listener)